gson/src/main/java/com/google/gson/FieldAdapterFactory.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2008 Google Inc.
```

Comment:
Sure. I copied the license lines from the bottom of google/gson's readme page, felt weird at first, but didn't think about it again. Maybe you guys should update it too.